azurerm/internal/identity/schema.go

```diff
@@ -14,10 +14,10 @@ const (
 type ExpandedConfig struct {
 	// Type is the type of User Assigned Identity, either `None`, `SystemAssigned`, `UserAssigned`
 	// or `SystemAssigned, UserAssigned`
-	Type                    string
-	PrincipalId             *string
-	TenantId                *string
-	UserAssignedIdentityIds *[]string
+	Type                    string   `tfschema:"type"`
+	PrincipalId             string   `tfschema:"principal_id"`
+	TenantId                string   `tfschema:"tenant_id"`
```

Comment:
Whilst this is fine, some of the other service packages (shown below) are expecting `*string` and `*[]string` - so we'll need to update those - I believe this'll be fixed by a rebase?